src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelTrace.cs

```diff
@@ -39,6 +40,7 @@ static KestrelTrace()
             // ConnectionWrite: Reserved: 11
             // ConnectionWriteCallback: Reserved: 12
             // ApplicationError: Reserved: 13 - LoggerMessage.Define overload not present 
+            _connectionDisconnectedWrite = LoggerMessage.Define<long, int>(LogLevel.Debug, 14, @"Connection id ""{ConnectionId}"" write of ""{count}"" bytes to disconnected client.");
         }
```

Comment:
Need some more clarity around the last parameter `Exception` parameter to logging as it does seem to come out. Adding one that takes an exception parameter.